observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,23 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6-2_{context}"]
+== Network Observability Operator 1.6.2
+The following advisory is available for the Network Observability Operator 1.6.2:
+
+* link:https://access.redhat.com/errata/RHSA-2024:6618[2024:6618 Network Observability Operator 1.6.2]
+
+[id="network-observability-operator-CVE-1-6-2_{context}"]
+=== CVEs
+* link:https://access.redhat.com/security/cve/CVE-2024-24791[CVE-2024-24791]
+
+[id="network-observability-operator-1-6-2-bug-fixes_{context}"]
+=== Bug fixes
+* When the secondary interface support was added, there was a need to iterate multiple times to register the per network namespace with the netlink to learn about interface notifications. At the same time, unsuccessful handlers caused a leaking file descriptor because with TCX hook, unlike TC, handlers needed to be explicitly removed when the interface went down. Now, there is no longer leaking file descriptors when creating and deleting pods. (link:https://issues.redhat.com/browse/NETOBSERV-1805[*NETOBSERV-1805*])
+
+=== Known issues
+This was a compatibility issue with console plugins that would have prevented Network Observability from being installed on future versions of an {product-title} cluster. By upgrading to 1.6.2, the compatibility issue is resolved and Network Observability can be installed as expected. (link:https://issues.redhat.com/browse/NETOBSERV-1737[*NETOBSERV-1737*])
```

Comment:
```suggestion
There was a compatibility issue with console plugins that would have prevented Network Observability from being installed on future versions of an {product-title} cluster. By upgrading to 1.6.2, the compatibility issue is resolved and Network Observability can be installed as expected. (link:https://issues.redhat.com/browse/NETOBSERV-1737[*NETOBSERV-1737*])
```

Reply:
Ok updated that and also changed the errata link per https://redhat-internal.slack.com/archives/C02939DP5L5/p1726211038071449?thread_ts=1725978135.396999&cid=C02939DP5L5